### DISCUSSION TITLE
Sub-path routing with country and language

### DISCUSSION THREAD

**Discussion by noidar** [i18next/next-i18next#1794] [Q&A]
I need to implement a translation soluition for a Nextjs project that should handle http://localhost:3000/de/ch/slugname
language:de country:ch
instead I am receiving { country:'slugname ', language:'ch'}
Here is an example of what I tried so far: https://github.dev/noidar/next-i18next-sub-path-routing-test

Is there any solution? Thanks.

**Comment by adrai** [comment]
Seems Next.js does not support this: https://nextjs.org/docs/advanced-features/i18n-routing

**Reply by noidar** [reply]
Responded here: https://github.com/vercel/next.js/discussions/36397.
Thanks.
